charts/refarch-gateway/README.md

```
@@ -1,3 +1,13 @@
 # RefArch gateway chart
 
 Helm chart for the [refarch-gateway](https://github.com/it-at-m/refarch).
+
+## Configuration
+
+### Memory limit change
+
+**Warning:** When `resources.limit.memory` is changed, `javaHeapMaxMemRatio` (mapped to `JAVA_MAX_MEM_RATIO`) needs to 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)